src/parseTools.js

```diff
@@ -873,8 +873,8 @@ function makeStructuralReturn(values) {
 }
 
 function makeThrow(what) {
-  if (ASSERTIONS && DISABLE_EXCEPTION_CATCHING == 1) {
-    what += ' + " - Exception catching is disabled, this exception cannot be caught. Compile with -s DISABLE_EXCEPTION_CATCHING=0 or DISABLE_EXCEPTION_CATCHING=2 to catch."';
+  if (ASSERTIONS && DISABLE_EXCEPTION_CATCHING) {
+    what += ' + " - Exception catching is disabled, this exception cannot be caught. Compile with -s NO_DISABLE_EXCEPTION_CATCHING or -s EXCEPTION_CATCHING_ALLOWED=[..] to catch."';
```

Comment:
Do we have a setting called `NO_DISABLE_EXCEPTION_CATCHING` somewhere?

Reply:
All settings can be prefixed with `NO_` to set them to zero as an alternative to `=0`.   

e.g. `-s NO_EXIT_RUNTIME` is the same as `-s EXIT_RUNTIME=0`

Personally I think this option reads better that way but I can revert this part if you prefer.

Reply:
No I didn't know there is always a matching `NO_` option for every setting. Thanks for letting me know.